tasks/test-server.js

```diff
@@ -3,12 +3,12 @@ const mocha = require('./base/mocha');
 const testServerPrereq = require('./test-server-prereq');
 
 const startMocha = (cb) => mocha(
-  cb,
-  ['src/server/scripts/**/*.js', 'src/shared/scripts/**/*.js'],
+  cb, ['src/server/scripts/**/*.js', 'src/shared/scripts/**/*.js'],
   ['spec/server/helpers/index.js', 'spec/shared/**/*.spec.js', 'spec/server/**/*.spec.js']
 );
-startMocha.displayName = 'Running Mocha on ./spec/**';
+startMocha.displayName = 'test-server';
```

Comment:
This displayName isn't used directly so it shouldn't need to be test-server. Better to leave it as it was.